docs/_content/standaard/standaard.md

```diff
@@ -235,6 +235,12 @@ controleren van alle gerelateerde informatieobjecten. Deze MAG NIET `null` zijn,
 maar MOET `true` of `false` zijn. Indien dit niet het geval is, dan dient het
 ZRC een validatiefout te genereren met code `indicatiegebruiksrecht-unset`.
 
+#### Heropenen zaak
+
+Bij het heropenen van een zaak MOET de client een andere status toevoegen aan de zaak dan een eindstatus. 
+Tevens MOET de client de 'Zaak.archiefactiedatum' leeg maken en het resultaat verwijderen. Het verwijderen van een resultaat MOET in de audit trail zichtbaar zijn.
+Dit betekent dat bij het weer afsluiten van een zaak, opnieuw een resultaat bij de zaak gezet moet worden en op basis daarvan de 'Zaak.archiefactiedatum' opnieuw berekend wordt.
```

Comment:
```suggestion
Dit betekent dat bij het weer afsluiten van een zaak, opnieuw een resultaat bij de zaak gezet MOET worden en op basis daarvan de 'Zaak.archiefactiedatum' opnieuw berekend wordt.
```

Reply:
Zou ik weghalen. Het zetten van een eindstatus kan ALLEEN als je een `Resultaat` hebt gezet. Dit volgt dus uit het ander. Als er al een `Resultaat` is zou deze zin ook niet kloppen.